sync/class.jetpack-sync-module-updates.php

```diff
@@ -112,26 +112,73 @@ public function update_core( $new_wp_version ) {
 
 	}
 
-	public function get_update_checksum( $value ) {
-		// Create an new array so we don't modify the object passed in.
-		$a_value = (array) $value;
-		// ignore `last_checked`
-		unset( $a_value['last_checked'] );
-		unset( $a_value['checked'] );
-		unset( $a_value['version_checked'] );
-		if ( empty( $a_value['updates'] ) ) {
-			unset( $a_value['updates'] );
-		}
 
-		if ( empty( $a_value ) ) {
+	public function get_update_checksum( $update, $transient ) {
+		$updates = array();
+		$no_updated = array();
+		switch ( $transient ) {
+			case 'update_plugins':
+				if ( ! empty( $update->response ) ) {
+					foreach ( $update->response as $plugin_slug => $response ) {
+						if ( ! empty( $plugin_slug ) && isset( $response->new_version ) ) {
+							$updates[] = array( $plugin_slug => $response->new_version );
+						}
+					}
+				}
+				if ( ! empty( $update->no_update ) ) {
+					$no_updated = array_keys( $update->no_update );
+				}
+
+				if ( ! isset( $no_updated[ 'jetpack/jetpack.php' ] ) && isset( $updates[ 'jetpack/jetpack.php' ] ) ) {
```

Comment:
what comes inside `no_update`?

Reply:
```
[no_update] => Array
        (
            [add-from-server/add-from-server.php] => stdClass Object
                (
                    [id] => w.org/plugins/add-from-server
                    [slug] => add-from-server
                    [plugin] => add-from-server/add-from-server.php
                    [new_version] => 3.3.3
                    [url] => https://wordpress.org/plugins/add-from-server/
                    [package] => https://downloads.wordpress.org/plugin/add-from-server.3.3.3.zip
                    [icons] => Array
                        (
                        )

                    [banners] => Array
                        (
                        )

                    [banners_rtl] => Array
                        (
                        )

                )

            [akismet/akismet.php] => stdClass Object
                (
                    [id] => w.org/plugins/akismet
                    [slug] => akismet
                    [plugin] => akismet/akismet.php
                    [new_version] => 4.0.2
                    [url] => https://wordpress.org/plugins/akismet/
                    [package] => https://downloads.wordpress.org/plugin/akismet.4.0.2.zip
                    [icons] => Array
```

Reply:
could we just ignore it when calculating the checksum?